app-scroll-effects/README.md

```diff
@@ -1,35 +1,75 @@
-## Polymer.AppScrollEffectsBehavior
+## Scroll effects
 
-`Polymer.AppScrollEffectsBehavior` provides an interface that allows to consume scrolls events.
-Effects are installed in the `Polymer.AppLayout.scrollEffects` namescape.
+`Polymer.AppScrollEffectsBehavior` provides an interface that allows to consume scrolls effects.
 
-A few effects are defined in `app-scroll-effects.html`, but, by default, no effect is imported.
-Thus, users need to explicitly import the dependency.
+### Importing the app-layout effects
 
-Elements that consume this behavior are required to override the folllowing methods:
+app-layout provides a set of scroll effects that can be used by explicitly importing `app-scroll-effects.html`:
 
-* `_updateScrollState`
-* `isOnScreen`
-* `isContentBelow`
-* `setup`
+```html
+<link rel="import" href="/bower_components/app-layout/app-scroll-effects/app-scroll-effects.html">
+```
 
-## Installing effects
+### Consuming effects
 
-Effects can be installed via the `effects` property. For example:
+Effects can be consumed via the `effects` property. For example:
 
 ```html
-<x-element effects="waterfall"></x-element>
+<app-header effects="waterfall"></app-header>
+```
+
+### Creating scroll effects
+
+You may want to create a custom scroll effect if you need to modify the CSS of an element
+based on the scroll position.
+
+A scroll effect definition is an object with a `setUp()`, `tearDown()` and `run()` function.
```

Comment:
...with `setUp()`, `tearDown()` and `run()` functions.

(Or replace functions with methods, but make sure it's plural)